README.md

```diff
@@ -1,127 +1,187 @@
 pip install dgl==2.3.0 -f https://data.dgl.ai/wheels/torch-2.3/repo.html
-#GPU
-pip install torch==2.3.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121
+
+# Or, to run on GPU use
+pip install torch==2.3.0 --index-url https://download.pytorch.org/whl/cu121
 pip install dgl==2.3.0+cu121 -f https://data.dgl.ai/wheels/torch-2.3/cu121/repo.html
 
 pip install graphstorm
 ```
 
-### Run GraphStorm with OGB datasets
+### Distributed training
 
-**Note**: we assume users have setup a GraphStorm standalone environment following the [Setup GraphStorm with pip Packages](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-with-pip-packages) instructions. And users have git cloned the GraphStorm source code into the `/graphstorm/` folder to use some complimentatry tools.
+To run GraphStorm in a distributed environment, we recommend using [Amazon SageMaker AI](https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html) to avoid having to manage cluster infrastructure. See our
+[SageMaker AI setup documentation](https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/distributed/sagemaker.html) to get started with distributed GNN training.
 
-**Node classification on OGB arxiv graph**
-First, use the below command to download the [OGB arxiv](https://ogb.stanford.edu/docs/nodeprop/#ogbn-arxiv) data and process it into a DGL graph for the node classification task.
+## Quick start
 
-```
-python /graphstorm/tools/gen_ogb_dataset.py --savepath /tmp/ogbn-arxiv-nc/ --retain-original-features true
-```
 
-Second, use the below command to partition this arxiv graph into a distributed graph that GraphStorm can use as its input.
+After installing GraphStorm and its requirements in your local environment as shown above, you can clone the GraphStorm repository to follow along the quick start examples:
 
+```bash
+git clone https://github.com/awslabs/graphstorm.git
+# Switch to the graphstorm repository root
+cd graphstorm
 ```
-python /graphstorm/tools/partition_graph.py --dataset ogbn-arxiv \
-                                            --filepath /tmp/ogbn-arxiv-nc/ \
-                                            --num-parts 1 \
-                                            --num-trainers-per-machine 4 \
-                                            --output /tmp/ogbn_arxiv_nc_train_val_1p_4t
-```
 
-GraphStorm training relies on ssh to launch training jobs. The GraphStorm standalone mode uses ssh services in port 22.
+### Node Classification on OGB arxiv graph
+
+This example demonstrates how to train a model to classify research papers in the OGB arxiv citation network. Each node represents a paper with a 128-dimensional feature vector, and the task is to predict the paper's subject area.
+
+First, download the [OGB arxiv](https://ogb.stanford.edu/docs/nodeprop/#ogbn-arxiv) data and process it into a DGL graph for the node classification task.
```

Comment:
Do we need to call `python /graphstorm/tools/gen_ogb_dataset.py --savepath /tmp/ogbn-arxiv-nc/ --retain-original-features true` ?

Reply:
In my testing of the quick start guide the below command is enough https://graphstorm.readthedocs.io/en/latest/tutorials/quick-start.html#download-and-partition-ogb-arxiv-data